packages/services/f-braze-adapter/src/services/_tests/BrazeDispatcher.node.test.js

```diff
@@ -22,8 +22,8 @@ describe('BrazeDispatcher › node', () => {
         expect.assertions(2);
 
         try {
-            const Dispatcher = new BrazeDispatcher(0);
-            await Dispatcher.configure({
+            // eslint-disable-next-line no-unused-vars
+            const Dispatcher = new BrazeDispatcher({
```

Comment:
If you're expecting the throw to happen within the constructor, you can just return, as you're expecting the assertions above:
```suggestion
            return new BrazeDispatcher({
```

Reply:
awesome thanks

Reply:
actually eslint then complains about `Expected to return a value at the end of async arrow function` if I remove it so can't win either way